inspire/base/templates/format/record/Inspire_Default_HTML_general_macros.tpl

```diff
@@ -17,8 +17,8 @@
 # 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
 #}
 
-{% macro render_author_names(author) %}
-  <a{% if author.affiliations|length > 0 %}
+{% macro render_author_names(author, show_affiliation) %}
```

Comment:
This refers specifically to "hover" affiliations. These should not be shown in the modal popup, since the affiliations will be already displayed in parentheses next to the author. Therefore, there is no need to display them on hover, in the modal window.